accounting/localizations/mexico.rst

```diff
@@ -625,3 +625,12 @@ FAQ
 .. _`official information here`: http://www.sat.gob.mx/fichas_tematicas/declaraciones_informativas/Paginas/declaracion_informativa_terceros.aspx
 .. _`Certificate`: ../../_static/files/certificate.cer
 .. _`Certificate Key`: ../../_static/files/certificate.key
+
+.. tip::
+   **Solution:** CFDI Payment terms (método de pago) of PPD or PUE (PUE – Pago en una sola exhibición and PPD – 
```

Comment:
Do not put it at the end of the file, after the external references. Add it in the appropriate section. If there is no proper section related to it, it is probably worth adding it.
Also probably worth adding a reference to be able to link to it.